### ISSUE TITLE
Node NotReady Discovered runtime cgroups name: /system.slice/docker.service

### ISSUE THREAD

**Issue by HelenaZheng** [kubernetes/kubernetes#85880]
**What happened**:
One node in k8s cluster not ready.

**What you expected to happen**:
What triggered the problem？How can i reproduce it ? How to avoid the problem happen again.

**Anything else we need to know?**:
First，I saw status of kubelet after I receive the alarm. Kubelet service is dead.
Second, I went to find the logs of journalcal since my kubelet installed by binary executable in systemctl. Then I saw the log before kubelet stopped:
Dec 03 20:30:06 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [30]waiting for 65 commands to complete for scsi0
Dec 03 20:30:11 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [35]waiting for 65 commands to complete for scsi0
Dec 03 20:30:12 b-dok1-ps-013 kubelet[206517]: I1203 20:30:12.198232  206517 container_manager_linux.go:428] [ContainerManager]: Discovered runtime cgroups name: /system.slice/docker.service
Dec 03 20:30:16 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [40]waiting for 65 commands to complete for scsi0
Dec 03 20:30:21 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [45]waiting for 65 commands to complete for scsi0
Dec 03 20:30:26 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [50]waiting for 65 commands to complete for scsi0
Dec 03 20:30:31 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [55]waiting for 65 commands to complete for scsi0
Dec 03 20:30:36 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [60]waiting for 65 commands to complete for scsi0
Dec 03 20:30:41 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [65]waiting for 65 commands to complete for scsi0
Dec 03 20:30:46 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [70]waiting for 65 commands to complete for scsi0
Dec 03 20:30:51 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [75]waiting for 65 commands to complete for scsi0
Dec 03 20:30:53 b-dok1-ps-013 kernel: INFO: task xfsaild/dm-0:14563 blocked for more than 120 seconds.
Dec 03 20:30:53 b-dok1-ps-013 kernel: "echo 0 > /proc/sys/kernel/hung_task_timeout_secs" disables this message.
Dec 03 20:30:53 b-dok1-ps-013 kernel: xfsaild/dm-0    D ffff966172f04100     0 14563      2 0x00000000
Dec 03 20:30:53 b-dok1-ps-013 kernel: Call Trace:
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f967bc9>] schedule+0x29/0x70
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc061ebc6>] _xfs_log_force+0x1c6/0x2c0 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2d67b0>] ? wake_up_state+0x20/0x20
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc062adcc>] ? xfsaild+0x16c/0x6f0 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc061ecec>] xfs_log_force+0x2c/0x70 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc062ac60>] ? xfs_trans_ail_cursor_first+0x90/0x90 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc062adcc>] xfsaild+0x16c/0x6f0 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc062ac60>] ? xfs_trans_ail_cursor_first+0x90/0x90 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2c1c31>] kthread+0xd1/0xe0
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2c1b60>] ? insert_kthread_work+0x40/0x40
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f974c1d>] ret_from_fork_nospec_begin+0x7/0x21
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2c1b60>] ? insert_kthread_work+0x40/0x40
Dec 03 20:30:53 b-dok1-ps-013 kernel: INFO: task xfsaild/dm-5:26056 blocked for more than 120 seconds.
Dec 03 20:30:53 b-dok1-ps-013 kernel: "echo 0 > /proc/sys/kernel/hung_task_timeout_secs" disables this message.
Dec 03 20:30:53 b-dok1-ps-013 kernel: xfsaild/dm-5    D ffff96616af25140     0 26056      2 0x00000000
Dec 03 20:30:53 b-dok1-ps-013 kernel: Call Trace:
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2aa0de>] ? try_to_del_timer_sync+0x5e/0x90
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f967bc9>] schedule+0x29/0x70
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc061ebc6>] _xfs_log_force+0x1c6/0x2c0 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2d67b0>] ? wake_up_state+0x20/0x20
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc062adcc>] ? xfsaild+0x16c/0x6f0 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc061ecec>] xfs_log_force+0x2c/0x70 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc062ac60>] ? xfs_trans_ail_cursor_first+0x90/0x90 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc062adcc>] xfsaild+0x16c/0x6f0 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc062ac60>] ? xfs_trans_ail_cursor_first+0x90/0x90 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2c1c31>] kthread+0xd1/0xe0
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2c1b60>] ? insert_kthread_work+0x40/0x40
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f974c1d>] ret_from_fork_nospec_begin+0x7/0x21
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2c1b60>] ? insert_kthread_work+0x40/0x40
Dec 03 20:30:53 b-dok1-ps-013 kernel: INFO: task xfsaild/dm-7:26085 blocked for more than 120 seconds.
Dec 03 20:30:53 b-dok1-ps-013 kernel: "echo 0 > /proc/sys/kernel/hung_task_timeout_secs" disables this message.
Dec 03 20:30:53 b-dok1-ps-013 kernel: xfsaild/dm-7    D ffff96616e34c100     0 26085      2 0x00000000
Dec 03 20:30:53 b-dok1-ps-013 kernel: Call Trace:
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f967bc9>] schedule+0x29/0x70
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f9656a1>] schedule_timeout+0x221/0x2d0
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f5430ba>] ? queue_unplugged+0x2a/0xa0
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f300f72>] ? ktime_get_ts64+0x52/0xf0
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f96726d>] io_schedule_timeout+0xad/0x130
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2c2886>] ? prepare_to_wait_exclusive+0x56/0x90
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f967308>] io_schedule+0x18/0x20
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f543b73>] get_request+0x243/0x7d0
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2c2d00>] ? wake_up_atomic_t+0x30/0x30
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f546a3e>] blk_queue_bio+0xfe/0x400
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f544c87>] generic_make_request+0x147/0x380
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f544f30>] submit_bio+0x70/0x150
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc05f9413>] _xfs_buf_ioapply+0x2f3/0x460 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc05fb2a8>] ? xfs_buf_delwri_submit_buffers+0x128/0x230 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc05fafcc>] xfs_buf_submit+0x6c/0x220 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc05fb2a8>] xfs_buf_delwri_submit_buffers+0x128/0x230 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc05fc0c0>] ? xfs_buf_delwri_submit_nowait+0x10/0x20 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc062ac60>] ? xfs_trans_ail_cursor_first+0x90/0x90 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc05fc0c0>] xfs_buf_delwri_submit_nowait+0x10/0x20 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc062aebf>] xfsaild+0x25f/0x6f0 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffffc062ac60>] ? xfs_trans_ail_cursor_first+0x90/0x90 [xfs]
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2c1c31>] kthread+0xd1/0xe0
Dec 03 20:30:53 b-dok1-ps-013 kernel:  [<ffffffff9f2c1b60>] ? insert_kthread_work+0x40/0x40
Dec 03 20:30:56 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [80]waiting for 65 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 systemd-journal[1536]: Permanent journal is using 3.0G (max allowed 2.9G, trying to leave 4.0G free of 26.2G available → current limit 3.0G).
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [85]waiting for 65 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [90]waiting for 65 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [95]waiting for 65 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [100]waiting for 65 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [105]waiting for 65 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [110]waiting for 65 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [115]waiting for 65 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [120]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [125]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [130]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [135]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [140]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [145]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [150]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [155]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [160]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [165]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [170]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [175]waiting for 64 commands to complete for scsi0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: pending commands remain after waiting, will reset adapter scsi0.
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: resetting fusion adapter scsi0.
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: Waiting for FW to come to ready state
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: FW now in Ready state
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: Current firmware supports maximum commands: 4077         LDIO threshold: 0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: FW supports sync cache        : Yes
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: Init cmd return status SUCCESS for SCSI host 0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: firmware type        : Extended VD(240 VD)firmware
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: controller type        : MR(4096MB)
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: Online Controller Reset(OCR)        : Enabled
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: Secure JBOD support        : Yes
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: NVMe passthru support        : Yes
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: Interrupts are enabled and controller is OPERATIONAL for scsi:0
Dec 03 20:33:54 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: Reset successful for scsi0.
Dec 03 20:28:53 b-dok1-ps-013 kernel:  [<ffffffff9f3b56b4>] filemap_fdatawait_range+0x14/0x30
Dec 03 20:28:53 b-dok1-ps-013 kernel:  [<ffffffff9f3b7b16>] filemap_write_and_wait_range+0x56/0x90
Dec 03 20:28:53 b-dok1-ps-013 kernel:  [<ffffffffc05ff156>] xfs_file_fsync+0x66/0x1c0 [xfs]
Dec 03 20:28:53 b-dok1-ps-013 kernel:  [<ffffffff9f475c77>] do_fsync+0x67/0xb0
Dec 03 20:28:53 b-dok1-ps-013 kernel:  [<ffffffff9f475f60>] SyS_fsync+0x10/0x20
Dec 03 20:28:53 b-dok1-ps-013 kernel:  [<ffffffff9f974ddb>] system_call_fastpath+0x22/0x27
Dec 03 20:29:36 b-dok1-ps-013 kernel: sd 0:2:0:0: [sda] tag#3 task abort called for scmd(ffff96660a80c0c0)
Dec 03 20:29:36 b-dok1-ps-013 kernel: sd 0:2:0:0: [sda] tag#3 CDB: Write(16) 8a 00 00 00 00 01 f7 6e 0e e0 00 00 00 08 00 00
Dec 03 20:29:36 b-dok1-ps-013 kernel: sd 0:2:0:0: task abort: FAILED scmd(ffff96660a80c0c0)
Dec 03 20:29:36 b-dok1-ps-013 kernel: sd 0:2:0:0: target reset called for scmd(ffff9661756b6900)
Dec 03 20:29:36 b-dok1-ps-013 kernel: sd 0:2:0:0: [sda] tag#11 megasas: target reset FAILED!!
Dec 03 20:29:36 b-dok1-ps-013 kernel: sd 0:2:0:0: [sda] tag#186 Controller reset is requested due to IO timeout
                                      SCSI command pointer: (ffff96916f892f40)         SCSI host state: 5         SCS
Dec 03 20:29:36 b-dok1-ps-013 kernel: IO request frame:
                                              f1000000 00000000 00000000 751045c0 00600002 00000020 00000000 00004000
                                              00000000 00000010 00000000 00000000 00000000 00000000 00000000 01000000
                                              0000008a 5b5c0100 0000607e 00002000 00000000 00000000 00000000 00000000
                                              00140012 00000010 31c3ed60 00000000 00000020 00000000 00020000 00000004
                                              6fa18000 00000006 00001000 00000000 43f79000 00000015 00001000 00000000
                                              d48ac000 00000020 00001000 00000000 d542e000 0000000c 00001000 40000000
                                              00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
                                              00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
Dec 03 20:29:36 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [ 0]waiting for 65 commands to complete for scsi0
Dec 03 20:29:41 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [ 5]waiting for 65 commands to complete for scsi0
Dec 03 20:29:46 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [10]waiting for 65 commands to complete for scsi0
Dec 03 20:29:51 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [15]waiting for 65 commands to complete for scsi0
Dec 03 20:29:56 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [20]waiting for 65 commands to complete for scsi0
Dec 03 20:30:01 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [25]waiting for 65 commands to complete for scsi0
Dec 03 20:30:01 b-dok1-ps-013 systemd[1]: Started Session 386467 of user root.
Dec 03 20:30:01 b-dok1-ps-013 systemd[1]: Started Session 386466 of user root.
Dec 03 20:30:01 b-dok1-ps-013 systemd[1]: Started Session 386468 of user root.
Dec 03 20:30:01 b-dok1-ps-013 CROND[455859]: (root) CMD (/bin/bash /etc/titanagent/agent_update_exception.sh >> /var/log/titanagent/check.o.log 2>> /var/log/titanagent/check.e.log)
Dec 03 20:30:01 b-dok1-ps-013 CROND[455860]: (root) CMD (/usr/lib64/sa/sa1 1 1)
Dec 03 20:30:01 b-dok1-ps-013 CROND[455861]: (root) CMD (/bin/bash /etc/titanagent/agent_monitor.sh >> /var/log/titanagent/edog.o.log 2>> /var/log/titanagent/edog.e.log)
Dec 03 20:30:06 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [30]waiting for 65 commands to complete for scsi0
Dec 03 20:30:11 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [35]waiting for 65 commands to complete for scsi0
Dec 03 20:30:12 b-dok1-ps-013 kubelet[206517]: I1203 20:30:12.198232  206517 container_manager_linux.go:428] [ContainerManager]: Discovered runtime cgroups name: /system.slice/docker.service
Dec 03 20:30:16 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [40]waiting for 65 commands to complete for scsi0
Dec 03 20:30:21 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [45]waiting for 65 commands to complete for scsi0
Dec 03 20:30:26 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [50]waiting for 65 commands to complete for scsi0
Dec 03 20:30:31 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [55]waiting for 65 commands to complete for scsi0
Dec 03 20:30:36 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [60]waiting for 65 commands to complete for scsi0
Dec 03 20:30:41 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [65]waiting for 65 commands to complete for scsi0
Dec 03 20:30:46 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [70]waiting for 65 commands to complete for scsi0
Dec 03 20:30:51 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [75]waiting for 65 commands to complete for scsi0

**Environment**:
- Kubernetes version (use `kubectl version`):
1.11.2
- Cloud provider or hardware configuration:
Physical machine in company‘s IDC
’
- OS (e.g: `cat /etc/os-release`):
NAME="Red Hat Enterprise Linux Server"
VERSION="7.6 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.6"
PRETTY_NAME="Red Hat Enterprise Linux Server 7.6 (Maipo)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.6:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.6
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="7.6"

- Kernel (e.g. `uname -a`):
Linux b-dok1-ps-013 3.10.0-957.el7.x86_64 #1 SMP Thu Oct 4 20:48:51 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

**Comment by HelenaZheng** [comment]
/sig node
/sig Architecture
@Derek Carr

**Comment by mars64** [comment]
Stumbled here researching cgroup configuration. I do not believe this is a Kubernetes or cgroups problem. 

```
container_manager_linux.go:428] [ContainerManager]: Discovered runtime cgroups name: /system.slice/docker.service
```

This line simply means the Kubelet was correctly able to read cgroup configuration.

```
Dec 03 20:30:16 b-dok1-ps-013 kernel: megaraid_sas 0000:5a:00.0: [40]waiting for 65 commands to complete for scsi0
```

This line means your storage had some kind of hiccup and didn't come up properly. You should investigate storage logs for this timeframe. Possible disk full scenario as per: 

```
Dec 03 20:33:54 b-dok1-ps-013 systemd-journal[1536]: Permanent journal is using 3.0G (max allowed 2.9G, trying to leave 4.0G free of 26.2G available → current limit 3.0G).
```
